### PR TITLE
FAD-5155 Revised metric denominator - UI

### PR DESCRIPTION
**Pull request by rajumsys**
Todo
- update docs/references 
 -- https://www.sparkpost.com/docs/tech-resources/sparkpost-event-metrics-definitions/#sparkpost-metrics-definitions
-- https://developers.sparkpost.com/api/metrics.html#header-metrics-api-deliverability-glossary
- accepted page

### CODE REVIEW

File: src/pages/reports/engagement/components/EngagementSummary.js [hunk 1]
```diff
@@ -27,16 +27,16 @@ export function EngagementSummary({ accepted = 0, clicks = 0, filters, loading,
     <Panel className={styles.EngagementSummary}>
       <Grid>
         <Grid.Column xs={12} md={3} xl={2}>
-          <h1>< Percent value={safeRate(opens, targeted)} /></h1>
+          <h1>< Percent value={safeRate(opens, sent)} /></h1>
           <h6>Unique Open Rate</h6>
         </Grid.Column>
         <Grid.Column xs={12} md={3} xl={2}>
-          <h1>< Percent value={safeRate(clicks, targeted)} /></h1>
+          <h1>< Percent value={safeRate(clicks, sent)} /></h1>
```
Comment (rajumsys) [inline src/pages/reports/engagement/components/EngagementSummary.js]: denominator for click/open will be accepted

File: src/pages/reports/engagement/components/async/EngagementChart.js [hunk 2]
```diff
@@ -69,18 +69,18 @@ export default function EngagementChart({ accepted = 0, clicks = 0, loading, ope
...
             <ReferenceArea
               x1="Accepted" x2="Unique Confirmed Opens"
-              percentage={formatPercent(safeRate(opens, targeted))}
+              percentage={formatPercent(safeRate(opens, accepted))}
               shape={PercentLabel}
             />
             <ReferenceArea
               x1="Unique Confirmed Opens" x2="Unique Clicks"
-              percentage={formatPercent(safeRate(clicks, targeted))}
+              percentage={formatPercent(safeRate(clicks, accepted))}
```
Comment (jonambas) [inline src/pages/reports/engagement/components/async/EngagementChart.js]: i'm not sure these percents are correct, I thought there were supposed to be drop off percentages from the previous metric.

cc @bkemper

File: src/pages/reports/accepted/components/TopLevelMetrics.js [hunk 3]
```diff
@@ -42,11 +42,11 @@ const TopLevelMetrics = ({ filters, metrics, aggregates }) => {
   return (
     <div>
       <MetricsSummary
-        rateValue={safeRate(count_accepted, count_targeted)}
+        rateValue={safeRate(count_accepted, count_sent)}
         rateTitle='Accepted Rate'
-        secondaryMessage={`${count_sent.toLocaleString()} messages were sent.`}
+        secondaryMessage={`${count_targeted.toLocaleString()} messages were targeted.`}
```
Comment (jonambas) [inline src/pages/reports/accepted/components/TopLevelMetrics.js]: tbh, you can probably just remove this all together. I don't see much value in leaving this in

Comment (rajumsys) [reply src/pages/reports/accepted/components/TopLevelMetrics.js]: yeah, i thought same. i'll remove it.

File: src/pages/reports/engagement/components/async/EngagementChart.js (hunk 2 above)
Comment (rajumsys) [reply src/pages/reports/engagement/components/async/EngagementChart.js]: so you're saying like unique click should be percentage in terms of unique open? i guess that's not what it is even in old UI. 

screen shot 2017-12-26 at 11 55 00 am


`9231/267067 * 100 = 3.46%`
here `267067` is the count_targeted which in this PR I replaced with `count_accepted`

Comment (jonambas) [reply src/pages/reports/engagement/components/async/EngagementChart.js]: Yeah no need to change, was just a comment

**Comment by jonambas** [comment]
in `helpers/acceptedjs` you can remove `'count_targeted',` from `getAcceptedMetrics`

File: src/pages/reports/engagement/components/async/EngagementChart.js (hunk 2 above)
Comment (bkemper) [reply src/pages/reports/engagement/components/async/EngagementChart.js]: @jonambas I thought that as well, but take a look at Ewan's comments in #249.

**Pull request merged**